cukinia

```diff
@@ -75,6 +75,19 @@ while getopts ":hvo:f:-:" opt; do
 done
 shift $((OPTIND-1))
 
+#init logging
+if [ -n $__log_format ]
```

Comment:
use "" for string testing.
cukinia style uses` ; then` on the same line as `if`